index.js

```diff
@@ -18,16 +18,20 @@ exports.communicate = async (url, methodName, message, options = {}) => {
     })
 }
 
-const createSoapClient = async (url, options) => {
-    const soapOptions = {
+exports.buildSoapOptions = (options) => {
+    return {
         escapeXML: options.escapeXML === true,
         returnFault: true,
         disableCache: true,
-        forceSoap12Headers: true,
+        forceSoap12Headers: options.forceSoap12Headers === undefined ? true : options.forceSoap12Headers,
```

Comment:
We can clean this up by writing:

`forceSoap12Headers: options.forceSoap12Headers || true`